.github/workflows/lint.yml

```diff
@@ -0,0 +1,46 @@
+name: Lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - branch-[0-9]+.[0-9]+
+
+permissions:
+  contents: read
+
```

Comment:
```suggestion
concurrency:
  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
  cancel-in-progress: true

defaults:
  run:
    shell: bash --noprofile --norc -exo pipefail {0}

env:
  PIP_EXTRA_INDEX_URL: https://download.pytorch.org/whl/cpu
```

Reply:
- concurrency: to kill unnecessary jobs
- shell: for better logs
- PIP_EXTRA_INDEX_URL: to install pytorch faster